content/docs/learning-guide/tutorials/physx/wind-provider.md

```diff
@@ -0,0 +1,103 @@
+
+1. Add a PhysX Force Region component to the entity. This component creates the wind force. To configure the component, do the following:
+
+1. Choose the {{< icon "add.svg" >}} **Add** button to add a new force.
+
+1. In the **Direction** property, set the **Y** component a value of `-1.0` and set the **Z** component to `0.0` to create a direction for the wind force.
+
+1. In the **Magnitude** property, set a value of `10.0` to create magnitude for the wind force.
+
+    The PhysX collider box displays cones representing the wind force direction.
+
+    {{< image-width src="/images/learning-guide/tutorials/physx/wind-force-region-setup.png" width="900" alt="Setting up a PhysX collider for a force region." >}}
+
+## Add a Cloth prefab
+
+1. To test the wind provider, add prefab with an NVIDIA Cloth mesh. In **Asset Browser**, navigate to `Gems\NvCloth\Assets\prefabs\Cloth`, locate the `cloth_locked_edge.prefab`, and then **drag** it into the viewport.
+
+1. Use the **Move** tool to place the cloth prefab. If you are using the **Local wind tag** property, you must place the cloth asset inside the PhysX Collider volume of the wind provider entity.
+
+    ![Positioning the cloth prefab in the wind provider entity.](/images/learning-guide/tutorials/physx/wind-provider-cloth-prefab.png)
+
+1. With the prefab in position, you can hide the wind provider entity. In Entity Outliner, click the **Show/Hide Entity** toggle in the column to the right of the wind provider entity to hide it.
+
+1. The `cloth_locked_edge.prefab` has the local wind property enabled on its **Cloth** component. This generates a local wind force from the Cloth component that overrides the wind force from the wind provider entity you created. The following steps disable the local wind force of the prefab so that you can view the results of the wind provider you created:
+        
+    1. In Entity Outliner, double click the {{< icon "prefab.svg" >}} **cloth_locked_edge** prefab to edit it in Focus Mode.
+
+    2. Click the {{< icon "entity.svg" >}} **cloth_locked_edge** child entity to select it.
+
+        ![Open the cloth prefab for editing in Focus Mode.](/images/learning-guide/tutorials/physx/edit-cloth-locked-edge-prefab.png)
+
+    3. With the {{< icon "entity.svg" >}} **cloth_locked_edge** entity selected, in Entity Inspector, in the Cloth component, expand the **Wind** property group and disable the **Enable local wind velocity** toggle.
+
+        {{< image-width src="/images/learning-guide/tutorials/physx/disable-local-wind-velocity.png" width="450" alt="Setting up a PhysX collider for a force region" >}}
+
+## Test the wind simulation
+
+Finally, you can test the simulation. With the **cloth_locked_edge** prefab open for editing in Focus Mode, do the following:
+
+1. At the top of the Cloth component, enable the **Simulate in editor** toggle. As the simulation begins, the cloth object might flip and stretch wildly, but it quickly settles into a breezy wind simulation.
```

Comment:
```suggestion
1. At the top of the Cloth component, turn on **Simulate in editor**. As the simulation begins, the cloth object might flip and stretch wildly, but it quickly settles into a breezy wind simulation.
```
For consistency with the previous edit of changing "disable" to "turn off", consider using "turn on" here (and dropping "toggle").